crates/ruff_python_formatter/src/trivia.rs

```diff
@@ -624,6 +681,37 @@ mod tests {
         test_case.assert_reverse_tokenization();
     }
 
+    #[test]
+    fn tricky_unicode_is_not_reversible_for_now() {
+        let source = "មុ";
```

Comment:
Hmm interesting. This is somewhat unexpected. I would expect that we consume the same text range

Reply:
i think this is fixed by the above conversation

but strings of non-identifiers (or any chars that would be considered `Other` like `555`) would still be parsed as `Other` followed by `Bogus`, and depending on direction, the `Other` would be first or last. This is the case currently on `main`, eg with `555` or even `aaa`.